### PR TITLE
Release notes for 1.22.0

### PR DESCRIPTION
**Pull request by krishi0408**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by SushrutHarness — APPROVED** [review]
Waiting on CDS-88190 otherwise it looks good!

**Comment by krishi0408** [comment]
> Waiting on CDS-88190 otherwise it looks good!

@SushrutHarness added the details for the same, please review. Thanks

**Review by SushrutHarness — DISMISSED** [review]
LGTM

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -49,6 +49,62 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
 ## January 2024
 
+### Version 1.22.0
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:52]:
```suggestion
### Version 1.22.3
```

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Pull request merged**